### PR TITLE
Make kwargs explicit for flow

### PR DESCRIPTION
**Pull request by joroeder**
This PR makes removes the kwargs of the Flow network class. See also #877

### CODE REVIEW

File: src/oemof/solph/flows/_flow.py [hunk 1]
```diff
@@ -120,7 +120,23 @@ class Flow(on.Edge):
...
+        positive_gradient=None,
+        negative_gradient=None,
+        full_load_time_max=None,
+        full_load_time_min=None,
+        variable_costs=0,
+        integer=None,
+        bidirectional=False,
+        investment=None,
+        nonconvex=None,
+        **kwargs,
```
Comment (joroeder) [inline src/oemof/solph/flows/_flow.py:138]: hier noch summed_max,

File: src/oemof/solph/flows/_flow.py [hunk 2]
```diff
@@ -135,59 +151,66 @@ def __init__(self, max=None, **kwargs):
...
+        self.bidirectional = bidirectional
+        
         need_nominal_value = [
             "fix",
             "full_load_time_max",
             "full_load_time_min",
             "min",
+            "max",
             # --- BEGIN: To be removed for versions >= v0.6 ---
             "summed_max",
```
Comment (joroeder) [inline src/oemof/solph/flows/_flow.py:196]: remove summed_max, summed_min

File: src/oemof/solph/flows/_flow.py (hunk 1 above)
Comment (joroeder) [reply src/oemof/solph/flows/_flow.py:138]: hier noch input=None, output=None

File: src/oemof/solph/flows/_flow.py [hunk 3]
```diff
@@ -135,59 +151,66 @@ def __init__(self, max=None, **kwargs):
...
         ]
 
-        need_nominal_value_dict = {
-            "max": max,
-        }
-
-        keys = [k for k in kwargs if k != "label"]
-
-        if "fixed_costs" in keys:
+        if "fixed_costs" in kwargs:
```
Comment (joroeder) [inline src/oemof/solph/flows/_flow.py:201]: remove

File: src/oemof/solph/flows/_flow.py [hunk 4]
```diff
@@ -135,59 +151,66 @@ def __init__(self, max=None, **kwargs):
...
             raise AttributeError(
                 "The `fixed_costs` attribute has been removed" " with v0.2!"
             )
 
-        if "actual_value" in keys:
+        if "actual_value" in kwargs:
```
Comment (joroeder) [inline src/oemof/solph/flows/_flow.py:206]: remove

File: src/oemof/solph/flows/_flow.py [hunk 5]
```diff
@@ -135,59 +151,66 @@ def __init__(self, max=None, **kwargs):
...
             raise AttributeError(
                 "The `actual_value` attribute has been renamed"
                 " to `fix` with v0.4. The attribute `fixed` is"
                 " set to True automatically when passing `fix`."
             )
 
-        if "fixed" in keys:
+        if "fixed" in kwargs:
```
Comment (joroeder) [inline src/oemof/solph/flows/_flow.py:213]: remove

**Comment by p-snft** [comment]
Leaving `positive_gradient` and `negative_gradient` as dicts with "ub" as the only allowed keyword adds some unneeded complication. I suggest to just name the limits directly.

**Comment by p-snft** [comment]
Error in Codacy is that no analysis takes place. I will merge because review will happen in #877.

**Pull request merged**